### PR TITLE
Added target for JHEF722ICM board

### PR DESCRIPTION
**Pull request by MaxKirillov**
Hey, thanks for running the project! Here is another board target (JHEMCU GF30F722-ICM).

### CODE REVIEW

File: src/main/target/JHEF722ICM/target.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+BUSDEV_REGISTER_SPI_TAG(busdev_icm42688, DEVHW_ICM42605, GYRO_SPI_BUS, GYRO1_CS_PIN, NONE, 0, DEVFLAGS_NONE, IMU_ICM42605_ALIGN);
+BUSDEV_REGISTER_SPI_TAG(busdev_icm42688_2, DEVHW_ICM42605, GYRO_SPI_BUS, GYRO2_CS_PIN, NONE, 1, DEVFLAGS_NONE, IMU_ICM42605_ALIGN);
+
+BUSDEV_REGISTER_SPI_TAG(busdev_bmi270, DEVHW_BMI270, GYRO_SPI_BUS, GYRO1_CS_PIN, NONE, 0, DEVFLAGS_NONE, IMU_BMI270_ALIGN);
+BUSDEV_REGISTER_SPI_TAG(busdev_bmi270_2, DEVHW_BMI270, GYRO_SPI_BUS, GYRO2_CS_PIN, NONE, 1, DEVFLAGS_NONE, IMU_BMI270_ALIGN);
+
+timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM9,  CH2, PA3,  TIM_USE_PPM,                          0, 0),
+    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MC_MOTOR  | TIM_USE_FW_MOTOR, 0, 0),
```
Comment (mmosca) [inline src/main/target/JHEF722ICM/target.c]: TIM_USE_MC_* and TIM_USE_FW_* need to be replaced with TIM_USE_OUTPUT_AUTO.

This is a change included in INAV 7.

**Comment by mmosca** [comment]
Please update the timer definitions, as the target, as suplied will not build against inav 7 or master.

**Comment by mmosca** [comment]
Waiting for hardware to validate:

 * [ ]  Flash firmware
 * [ ]  Calibrate
 * [ ]  Orientation matches
 * [ ]  Gyro working
 * [ ]  Accel working
 * [ ]  Baro working
 * [ ]  Voltage correct
 * [ ]  Current correct
 * [ ]  UARTs working
 * [ ]  Camera working
 * [ ]  Video Out working
 * [ ]  OSD working
 * [ ]  LEDs working
 * [ ]  Buzzer working
 * [ ]  Motor outputs

File: src/main/target/JHEF722ICM/target.c (hunk 1 above)
Comment (MaxKirillov) [reply src/main/target/JHEF722ICM/target.c]: Thanks for pointing me to the issue. Seems I missed some updates) Now I have it successfully built locally.

File: src/main/target/JHEF722ICM/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+target_stm32f722xe(JHEF722ICM)
```
Comment (mmosca) [inline src/main/target/JHEF722ICM/CMakeLists.txt]: Since this is not comming from JHEMCU, can you add SKIP_RELEASES here?

**Comment by shota3527** [comment]
board received


- [ ] Flash firmware
- [ ]  Calibrate
- [ ]  Orientation matches
- [ ]  Gyro working
- [ ]  Accel working
- [ ]  Baro working
- [ ]  Voltage correct
- [ ]  Current correct
- [ ]  UARTs working
- [ ]  Camera working
- [ ]  Video Out working
- [ ]  OSD working
- [ ]  LEDs working
- [ ]  Buzzer working
- [ ]  Motor outputs

**Pull request closed without merging**